Hall_B/AIDAPT/aidapt_toolkit/utils/config_utils.py

```diff
@@ -5,5 +5,7 @@ def verify_config(config, required_keys):
             missing_keys.append(key)
 
     if missing_keys:
-        error_msg = f'Config is missing the following required keys: {", ".join(missing_keys)}'
-        raise KeyError(error_msg)
\ No newline at end of file
+        error_msg = (
+            f'Config is missing the following required keys: {", ".join(missing_keys)}'
```

Comment:
Maybe there's a better way to do this to avoid 3 lines for a relatively short string...